cfme/tests/networks/nuage/test_inventory.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from cfme.networks.provider.nuage import NuageProvider
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider], scope='module')
+]
+
+
+def test_tenant_details(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
+    sandbox = with_nuage_sandbox_modscope
+    tenant_name = sandbox['enterprise'].name
+    tenant = provider.cloud_tenants.get_by_name(tenant_name)
+
+    expected_stats = {
+        'num_cloud_subnets': 2,
+        'num_network_routers': 1,
+        'num_security_groups': 2,
+        'num_network_ports': 4
+    }
+    tenant.validate_stats(expected_stats)
```

Comment:
Please just pass the expected hash inline, no need to introduce `expected_stats` variable at all.